app-text/a2ps/a2ps-4.15.6.recipe

```diff
@@ -0,0 +1,100 @@
+SUMMARY="A Any to PostScript filter"
+DESCRIPTION="GNU a2ps is an Any to PostScript filter.  Of course it processes plain text files, \
+but also pretty prints quite a few popular languages.
+
+Its slogan is precisely: Do The Right Thing, which means that though it is highly configurable, \
+everything was made so that a novice user can do complicated PostScript manipulations.  For \
+instance, it has the ability to delegate the processing of some files to other filters (such as \
+groff, texi2dvi, dvips, gzip etc.), what allows a uniform treatment (n-up, page selection, duplex \
+etc.) of heterogeneous files."
+HOMEPAGE="https://www.gnu.org/software/a2ps/"
+COPYRIGHT="2024 Akim Demaille, Miguel Santana, and Reuben Thomas"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/gnu/a2ps/a2ps-$portVersion.tar.gz"
+CHECKSUM_SHA256="87ff9d801cb11969181d5b8cf8b65e65e5b24bb0c76a1b825e8098f2906fbdf4"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+GLOBAL_WRITABLE_FILES="
+	settings/a2ps-site.cfg keep-old
+	settings/a2ps.cfg keep-old
+	"
+
+PROVIDES="
+	a2ps$secondaryArchSuffix = $portVersion
+	cmd:a2ps$commandSuffix = $portVersion
+	cmd:a2ps_lpr_wrapper$commandSuffix = $portVersion
+	cmd:card$commandSuffix = $portVersion
+	cmd:composeglyphs$commandSuffix = $portVersion
+	cmd:fixps$commandSuffix = $portVersion
+	cmd:lp2$commandSuffix = $portVersion
+	cmd:ogonkify$commandSuffix = $portVersion
+	cmd:pdiff$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libgc$secondaryArchSuffix
+	lib:libpaper$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libgc$secondaryArchSuffix
+	devel:libgs$secondaryArchSuffix
+	devel:libpaper$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:convert
+	cmd:gcc$secondaryArchSuffix
+	cmd:groff
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:msgfmt$secondaryArchSuffix
+	cmd:perl
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:psnup
+	"
+
+defineDebugInfoPackage a2ps$secondaryArchSuffix \
+	$commandBinDir/a2ps
+
+BUILD()
+{
+	autoreconf -fi
+	runConfigure --omit-dirs "binDir sbinDir" ./configure \
+		--bindir=$commandBinDir
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+}
+
+TEST()
+{
+	# TOTAL: 23
+	# PASS:  3
+	# SKIP:  0
+	# XFAIL: 0
+	# FAIL:  20
+	# XPASS: 0
+	# ERROR: 0
+	# mostly "a2ps: unknown medium `A4'"
```

Comment:
This is probably due to the missing paperspecs config file coming from libpaper, which isn't available in the haikuporter chroot. If that's the case, see here for a workaround: https://github.com/haikuports/haikuports/blob/5d0772dffe88b2f875091aa661038311dac8e2d2/app-text/psutils/psutils-3.3.5.recipe#L73-L81

Reply:
Thanks! Will do another check once I'm done on Dolphin and friends. :)

Reply:
This worked 👍 22 passed out of 23 now!
Question, would it be needed to also apply this in BUILD()?